options/posix/generic/posix_locale.cpp

```diff
@@ -2,12 +2,15 @@
 #include <bits/ensure.h>
 
 locale_t newlocale(int category_mask, const char *locale, locale_t base) {
-	__ensure(!"Not implemented");
-	__builtin_unreachable();
+	// Due to all of the locale functions being stubs, the locale will not be used
```

Comment:
We should probably do some logging to say it's a stub.

Reply:
Sure